app/matplotlib.ipynb

```diff
@@ -123,21 +121,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "categories = [\"Category A\", \"Category B\", \"Category C\", \"Category D\"]\n",
     "sales = [350, 480, 210, 590]\n",
     "\n",
-    "# write your code here\n"
+    "plt.bar(categories, sales, color=\"red\")\n",
+    "\n",
+    "plt.title(\"Bar graph\")\n",
+    "plt.xlabel(\"x label\")\n",
+    "plt.ylabel(\"y label\")\n",
+    "plt.show()\n"
```

Comment:
The task requires rotating the x-ticks by 45 degrees. You can achieve this by using `plt.xticks(rotation=45)` after setting the labels.